hack/single-node/user-data.sample

```diff
@@ -26,14 +26,17 @@ coreos:
       content: |
         [Service]
         EnvironmentFile=/etc/environment
-        Environment=KUBELET_ACI=quay.io/coreos/hyperkube
-        Environment=KUBELET_VERSION=v1.3.0-alpha.5_coreos.0
+        Environment=KUBELET_ACI=quay.io/peanutbutter/hyperkube
+        Environment=KUBELET_VERSION=v1.3.0-alpha.5_derek.1
```

Comment:
I think we can drop the custom image for now. We can ship a release which has the checkpoint code, but doesn't use it. Then we can update this in the follow up PR (also should have an official `beta.1` soon which contains the hostip fix).